html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-syntax</title>
+  </head>
+  <body>
+    <img
+      src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png"
+      alt="Picture of Pikachu" />
+    <h1><strong>Pikachu</strong></h1>
```

Comment:
`<strong>` isn't needed here. By using a `h1`, it will already be bold, and it will already be read as the heading as the page, so it doesn't need any additional emphasis.

Reply:
copy h1-h6 tags add emphasis already no need for unnecessary <strong> elements